sp-docs/docs/work.rst

```diff
@@ -107,6 +107,12 @@ Ensure there are no spelling errors in the documentation:
 
    make spelling
 
+If you only want to check the existing output and do not want to build the HTML again, run the spelling check separately:
+
+.. code-block:: shell
+
+   make spellcheck 
```

Comment:
Good catch! I totally forgot to add that to the docs when I added the target. :)